pkg/sdk/custom_shapes.go

```diff
@@ -42,6 +42,7 @@ type customShapeInjector struct {
 // the SDKAPI object.
 func (h *Helper) InjectCustomShapes(sdkapi *ackmodel.SDKAPI) error {
 	injector := customShapeInjector{sdkapi}
+	h.InjectSimpleSchemaShapes(sdkapi)
```

Comment:
shall we declare this before injector? flow looks like:
-> we inject some custom structs
-> initialise the injector
-> inject list/map custom types.

Now i'm actually thinking should we move some of that logic to the customShapeInjector?